samples/spring-transfer-workflow/src/main/java/com/example/wallet/Balance.java

```diff
@@ -0,0 +1,4 @@
+package com.example.wallet;
+
+public record Balance(int value) {
```

Comment:
We can add the domain types inside the entities. Same story, all self-contained. 

Make it also easier to document.